cache/cache_test.go

```diff
@@ -8,17 +8,44 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestExpirationWithJitter(t *testing.T) {
+	// Since expirationWithJitter use random, to multiple time to ensure
+	// result is within expected boundary
```

Comment:
```suggestion
	/* Since expirationWithJitter randomizes the expirtaion time, and instructions take a random time already,
	we need to make room for tests.
	*/
```

Reply:
I'm not sure to understand both your and my sentence :D
I likely did a typo and wanted to write "Since expirationWithJitter use random, DO multiple time [...]" or "Since expirationWithJitter use random, test multiple time [...]".

What I wanted to said it that since the result is randomized, we can't test for exact value but only that result is within a range. Doing multiple tests ensure that all results are within the range.

What about this version ?:
```
// Since expirationWithJitter randomizes the expirtaion time, do test multiple times and check that result is within expected range
```

Reply:
Ah, gotcha! Yeah, the `room` I mentioned was more about testing to the limit since we don't know how much the instructions take. I get what you were saying now and your new comment makes sense to me.